_search-plugins/star-tree-index.md

```diff
@@ -39,26 +39,26 @@ The following image illustrates a standard star-tree index structure.
 
 ### Star nodes
 
-Star nodes are children of non-leaf nodes that contain preaggregated records for data split after dimension removal, aggregating metrics for rows containing dimensions with identical values. These aggregated documents are then appended to the end of star-tree documents. If a document does contain a dimension with identical values, it traverses through the star node.
+Star nodes contain the aggregated data of all the other nodes for a particular dimension, acting as "catch-all" node. When a star node is found in a dimension, that dimension is skipped during aggregation. This group all values of that dimension together. This allows a query to skip non-competitive nodes when fetching the aggregated value of a particular field. 
 
-The star-tree index structure diagram contains the following three examples demonstrating how a document does or does not traverse star-tree nodes (indicated by the `*` symbol in the diagram) during a `Term` query, based on the average request size of the query and whether the document contains matching dimensions:
+The star-tree index structure diagram contains the following three examples demonstrating how a query behaves when retrieving aggregations from nodes in the star-tree:
 
-- When the port equals `8443` and the status equals `200`. Because the status equals `200`, the query does not traverse through a star node, and the aggregated metric is stored at the end of a star-tree document.
-- When the status equals `200`. The query traverses through a star node in the `port` dimension because `port` is not present as part of the query.
-- When the port equals `5600`. The query traverses through a star node in the `status` dimension because `status` is not present as part of the query.
+- **Blue**: In a `terms` query searching for the average request size aggregation, the `port` equals `8443` and the status equals `200`. Because the query contains values in both the `status` and `port` dimensions, the query traverses through staus node `200` and returns the aggregations from child node `8443`.
+- **Green**: In a `term` query searching for the count of requests aggregation, the `status` equals `200`. Because the query only contains a value from the `status` dimension, the query traverses to the `200` nodes child star node, which contains the aggregated vale of all the `port` children nodes.
+- **Red**: In a `term` query searching for the computed average request size aggregation, the port equals `5600`. Because the query does not contain a value from the `status` dimension, the query traverses through a star node and returns the aggregated result from the `5600` child node.
```

Comment:
```suggestion
- **Red**: In a `term` query that searches for the computed average request size aggregation, the port equals `5600`. Because the query does not contain a value from the `status` dimension, the query traverses a star node and returns the aggregated result from the `5600` child node.
```